### PR TITLE
Update terraform-docs to v.0.13.0

### PR DESCRIPTION
**Pull request by kumoy**

### CODE REVIEW

**Review by lorengordon — COMMENTED** [review]
Can you update the changelog, and run `bumpversion minor`?

**Comment by lorengordon** [comment]
@kumoy your branch is out of date, causing conflicts for the version update... please update the branch from upstream, fixed any conflicts, and push again. let me know if you need a hand.

**Comment by kumoy** [comment]
@lorengordon I think I did it. Please review!

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.12.5
+
+**Released**: 2021.05.06
+
+**Commit Delta**: [Change from 0.12.5 release](https://github.com/plus3it/tardigrade-ci/compare/0.12.0..0.12.5)
```
Comment (kbalk) [inline CHANGELOG.md:11]: "0.12.0..0.12.5" -> "0.12.4..0.12.5"

Comment (kumoy) [reply CHANGELOG.md:11]: Thank you! Missed that :(

Comment (kbalk) [reply CHANGELOG.md]: It's rather rich of me to point out an error in a CHANGELOG.md file given how many mistakes I've made with that file in various repos.

File: .bumpversion.cfg [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.12.4
+current_version = 0.12.5
```
Comment (lorengordon) [inline .bumpversion.cfg:2]: This needs to be a minor release, as the change will be backwards-incompatible.
```suggestion
current_version = 0.13.0
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.12.5
```
Comment (lorengordon) [inline CHANGELOG.md:7]:
```suggestion
### 0.13.0
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
...
+
+**Released**: 2021.05.06
+
+**Commit Delta**: [Change from 0.12.5 release](https://github.com/plus3it/tardigrade-ci/compare/0.12.4..0.12.5)
```
Comment (lorengordon) [inline CHANGELOG.md:11]:
```suggestion
**Commit Delta**: [Change from 0.12.4 release](https://github.com/plus3it/tardigrade-ci/compare/0.12.4..0.13.0)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.13.0
+
+**Released**: 2021.05.06
+
+**Commit Delta**: [Change from 0.13.0 release](https://github.com/plus3it/tardigrade-ci/compare/0.12.4..0.13.0)
```
Comment (lorengordon) [inline CHANGELOG.md:11]: The link title is for the changes from the **prior** release...

```suggestion
**Commit Delta**: [Change from 0.12.4 release](https://github.com/plus3it/tardigrade-ci/compare/0.12.4..0.13.0)
```

Comment (kumoy) [reply CHANGELOG.md:11]: Ohhh...oops.

**Review by lorengordon — APPROVED** [review]
🚀

**Pull request merged**